Documentation/ceph-fs-mirror-crd.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FilesystemMirror CRD
+weight: 3500
+indent: true
+---
+
+# Ceph FilesystemMirror CRD
+
+Rook allows creation and updating fs-mirror daemon through the custom resource definitions (CRDs).
+CephFS images can be asynchronously mirrored between two Ceph clusters.
+CephFS will support asynchronous replication of snapshots to a remote CephFS file system via cephfs-mirror tool.
+Snapshots are synchronized by mirroring snapshot data followed by creating a snapshot with the same name (for a given directory on the remote file system) as the snapshot being synchronized.
+For more information about user management and capabilities see the [Ceph docs](https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#cephfs-mirroring).
+
+## Creating daemon
+
+To get you started, here is a simple example of a CRD to deploy an cephfs-mirror daemon.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemMirror
+metadata:
+  name: my-fs-mirror
+  namespace: rook-ceph
+```
```

Comment:
Are there no settings like the number of mirroring daemons to start? Or is only a single daemon supported?

Reply:
do we need to pass some tokens which contain remote information that the mirroring daemon will connect to?

Reply:
Yes, but this is coming in another PR, I'm waiting for ceph/ceph#39050 to merge so I can start the implementation.